docs/vlab/overview.md

```diff
@@ -71,6 +74,28 @@ INFO: /dev/kvm exists
 KVM acceleration can be used
 ```
 
+### Oras
+
+For convenience Hedgehog provides a script to install `oras`:
+
+```bash
+curl -fsSL https://i.hhdev.io/oras | bash
+```
+
+### Hhfab
+
+A github access token is required to download `hhfab`, please submit a ticket using the [Hedgehog Support Portal](https://support.githedgehog.com/). Once obtained, use the provided username and token to log into the Github container registry:
+
+```bash
+docker login ghcr.io --username provided_username --password provided_token
+```
+
+Hedgehog maintains a utility to install and configure VLAB, called `hhfab`. To install:
```

Comment:
> Hedgehog maintains a utility to install and configure VLAB, called `hhfab`.

I'd move that to the top of the subsection, before mentioning the token required to download this `hhfab` you mention :)

Also s/github/GitHub/